lib/matplotlib/tests/test_table.py

```diff
@@ -1,12 +1,16 @@
+import datetime
 import numpy as np
 import pytest
+from unittest.mock import MagicMock
+
```

Comment:
These belong in first section:
```suggestion
import datetime
from unittest.mock import MagicMock

import numpy as np
import pytest
```